apps/epic-web/src/emails/lesson-complete-email.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Body,
+  Container,
+  Head,
+  Html,
+  Preview,
+  Section,
+  Text,
+} from '@react-email/components'
+import * as React from 'react'
+import {Markdown} from '@react-email/markdown'
+
+export const LessonCompleteEmail = ({
```

Comment:
I kept these super simple, but you can be fancier with react-email. I like the plaintext vibe and the ability to use markdown like this is rad.